AnimatedWordCloud/Animator/TimelapseWordVector.py

```diff
@@ -138,3 +199,25 @@ def add_time_frame(self, timeframe: TimeFrame) -> None:
         """
 
         self.timeframes.append(timeframe)
+
+    def convert_from_dicts_list(
+        data: Iterable[Iterable[str, Dict[str, float]]]
+    ) -> TimelapseWordVector:
+        """
+        Convert from a list of dictionary of word and weight to TimelapseWordVector instance.
+
+        This is static conversion method.
+
+        :param Iterable[Iterable[str, Dict[str, float]]] data: list[(time_name, Dict[word, weight])]
+        :return: The TimelapseWordVector instance
+        :rtype: TimelapseWordVector
+        """
+
+        instance = TimelapseWordVector()
+
+        for word_weights in data:
+            instance.add_time_frame(
+                TimeFrame.convert_from_tup_dict(word_weights)
```

Comment:
TimeFrame().convert_from_tup_dict(word_weights)

Reply:
If "TimeFrame().convert_from_tup_dict(word_weights)" works well, you need not to change this line.